com.unity.render-pipelines.core/Editor/PostProcess/SRPLensFlareOverrideEditor.cs

```diff
@@ -0,0 +1,127 @@
+    [CustomEditor(typeof(SRPLensFlareOverride))]
+    public class SRPLensFlareOverrideEditor : Editor
+    {
+        SerializedProperty m_LensFlareData;
+        SerializedProperty m_Intensity;
+        SerializedProperty m_Scale;
+        SerializedProperty m_MaxAttenuationDistance;
+        SerializedProperty m_MaxAttenuationScale;
+        SerializedProperty m_DistanceAttenuationCurve;
+        SerializedProperty m_ScaleByDistanceCurve;
+        SerializedProperty m_AttenuationByLightShape;
+        SerializedProperty m_RadialScreenAttenuationCurve;
+        SerializedProperty m_UseOcclusion;
+        SerializedProperty m_OcclusionRadius;
+        SerializedProperty m_SamplesCount;
+        SerializedProperty m_OcclusionOffset;
+        SerializedProperty m_AllowOffScreen;
+
+        /// <summary>
+        /// Prepare the code for the UI
+        /// </summary>
+        public void OnEnable()
+        {
+            PropertyFetcher<SRPLensFlareOverride> entryPoint = new PropertyFetcher<SRPLensFlareOverride>(serializedObject);
+            m_LensFlareData = entryPoint.Find("m_LensFlareData");
+            m_Intensity = entryPoint.Find(x => x.intensity);
+            m_Scale = entryPoint.Find(x => x.scale);
+            m_MaxAttenuationDistance = entryPoint.Find(x => x.maxAttenuationDistance);
+            m_DistanceAttenuationCurve = entryPoint.Find(x => x.distanceAttenuationCurve);
+            m_MaxAttenuationScale = entryPoint.Find(x => x.maxAttenuationScale);
+            m_ScaleByDistanceCurve = entryPoint.Find(x => x.scaleByDistanceCurve);
+            m_AttenuationByLightShape = entryPoint.Find(x => x.attenuationByLightShape);
+            m_RadialScreenAttenuationCurve = entryPoint.Find(x => x.radialScreenAttenuationCurve);
+            m_UseOcclusion = entryPoint.Find(x => x.useOcclusion);
+            m_OcclusionRadius = entryPoint.Find(x => x.occlusionRadius);
+            m_SamplesCount = entryPoint.Find(x => x.sampleCount);
+            m_OcclusionOffset = entryPoint.Find(x => x.occlusionOffset);
+            m_AllowOffScreen = entryPoint.Find(x => x.allowOffScreen);
+        }
+
+        /// <summary>
+        /// Implement this function to make a custom inspector
+        /// </summary>
+        public override void OnInspectorGUI()
+        {
+            SRPLensFlareOverride lensFlareData = m_Intensity.serializedObject.targetObject as SRPLensFlareOverride;
+            bool attachedToLight = false;
+            bool lightIsDirLight = false;
+            Light light = null;
+            if (lensFlareData != null &&
+                (light = lensFlareData.GetComponent<Light>()) != null)
+            {
+                attachedToLight = true;
+                if (light.type == LightType.Directional)
+                    lightIsDirLight = true;
+            }
+
+            EditorGUI.BeginChangeCheck();
+            EditorGUILayout.BeginFoldoutHeaderGroup(false, "    General", EditorStyles.boldLabel);
```

Comment:
label should be in a style class to not search them in the logic and it should be localizable.
Please use either **L18n** or **EditorGUIUtility.TrTextContent**
Plus here you have some spacing done with space character. Can you change it to either use indentation or displace directly the drawing rect starting point?